src/screens/ContactScreen/index.js

```diff
@@ -18,6 +18,7 @@ import {Loading} from '../../components';
 import {Search} from './elements';
 import {userPopulate} from '../../service/users';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {ANONYMOUS} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ANONYMOUS` constant is a new addition. It seems to be used as a value to compare with the source screen to determine whether the user is anonymous or not. Ensure that the value of this constant correctly represents an anonymous user.
```diff
+ import {ANONYMOUS} from '../../hooks/core/constant';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->